src/modules/endpoint/DatabaseConnector.js

```diff
@@ -73,11 +83,15 @@ const mapToObject = (map: Map<string, string>) => {
 }
 
 class DatabaseConnector {
-  getContentPath (key: string, context: DatabaseContext): string {
+  getContentPath (key: string, context: DatabaseContext = null): string {
     if (!key) {
       throw Error('Key mustn\'t be empty')
     }
 
+    if (context == null) {
+      return `${CONTENT_DIR_PATH}/${key}.json`
```

Comment:
Maybe this should rather be excluded to a separate method, since it does not have to do anything with the `context`. Not sure though.

Reply:
I think it is fine as the json lies withing the `CONTENT_DIR_PATH`

Reply:
For consistency with the functions around it you should create a separate funciton